jrnl/install.py

```diff
@@ -75,6 +75,12 @@ def load_or_install_jrnl(alt_config_path):
         logging.debug("Reading configuration from file %s", config_path)
         config = load_config(config_path)
 
+        if config is None:
+            print(
+                f"Configuration file {config_path} is empty, now using default config"
+            )
```

Comment:
This error message should go to STD_ERR -- just need to add `file=sys.stderr` to the print args.

Reply:
Indeed better to direct this to the error output.